resources/views/organizations/create.blade.php

```diff
@@ -0,0 +1,134 @@
+                        <label for="number_of_employers">Number of employers:</label>
+                        <input type="text" id="number_of_employers" name="number_of_employers" value="{{ old('number_of_employers') }}">
+                        @error('number_of_employers')
+                        <div>
+                            {{ $message }}
+                        </div>
+                        @enderror
+                    </div>
+                    <div>
+                        <label for="website_url">Website:</label>
+                        <input type="text" id="website_url" name="website_url" value="{{ old('website_url') }}">
+                        @error('website_url')
+                        <div>
+                            {{ $message }}
+                        </div>
+                        @enderror
+                    </div>
+                    <div>
+                        <label for="facebook_url">Facebook:</label>
+                        <input type="text" id="facebook_url" name="facebook_url" value="{{ old('facebook_url') }}">
+                        @error('facebook_url')
+                        <div>
+                            {{ $message }}
+                        </div>
+                        @enderror
+                    </div>
+                    <div>
+                        <label for="linkedin_url">Linkedin:</label>
+                        <input type="text" id="linkedin_url" name="linkedin_url" value="{{ old('linkedin_url') }}">
+                        @error('linkedin_url')
+                        <div>
+                            {{ $message }}
+                        </div>
+                        @enderror
+                    </div>
+                    <div>
+                        <label for="instagram_url">Instagram:</label>
+                        <input type="text" id="instagram_url" name="instagram_url" value="{{ old('instagram_url') }}">
+                        @error('instagram_url')
+                        <div>
+                            {{ $message }}
+                        </div>
+                        @enderror
+                    </div>
+                    <div>
+                        <label for="youtube_url">YouTube:</label>
+                        <input type="text" id="youtube_url" name="youtube_url" value="{{ old('youtube_url') }}">
+                        @error('youtube_url')
+                        <div>
+                            {{ $message }}
+                        </div>
+                        @enderror
+                    </div>
+                    <div>
+                        <label for="twitter_url">Twitter:</label>
+                        <input type="text" id="twitter_url" name="twitter_url" value="{{ old('twitter_url') }}">
+                        @error('twitter_url')
+                        <div>
+                            {{ $message }}
+                        </div>
+                        @enderror
```

Comment:
Things like this are repeated for ten times here. Maybe you could move it to some component?